sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -18,7 +18,7 @@ internal CustomFormModel(Model_internal model)
             Status = model.ModelInfo.Status;
             RequestedOn = model.ModelInfo.CreatedDateTime;
             CompletedOn = model.ModelInfo.LastUpdatedDateTime;
-            Models = ConvertToSubmodels(model);
+            Submodels = ConvertToSubmodels(model);
```

Comment:
We should be consistent in casing with the type name.  Per [this guideline](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalizing-compound-words-and-common-terms), if we need to update the type name casing, let's do that.

Reply:
good point. I renamed the type